docs/accessibility/policies.md

```diff
@@ -0,0 +1,41 @@
+---
+permalink: /accessibility/policies/
+title: Accessibility policies
+sidenav:
+  - text: Decorative images
+    href: "#using-decorative-images"
+---
+[← Back to Login.gov Accessibility home]({% link accessibility.md %})
+
+## Using decorative images
+
+A decorative image does not add information to the content of a page. Unlike an informative image, a decorative image can be hidden from screen readers. To help prevent confusing, redundant and/or tedious experience, we at Login.gov have set decorative images to be hidden from screen readers.
+
+Examples (See visual screenshots below):
+
+- An icon with adjacent text
+- An illustration that makes the page more attractive, but is non-essential to understanding the content
+- A screenshot image with a caption
+
+If the image is decorative, the alt attribute (sometimes referred to as "alt tag") would be empty (Example: `alt=""`), so the user knows that it is intentionally left empty. An empty alt attribute communicates that an image is decorative.
+
+Avoid ommitting the alt attribute altogether. If the alt attribute is not used, a screen reader will read out the full filename to the user, which is not an accessible or good user experience.
+
+If an image is informative, we add a description with a caption, alt attribute, title attribute, or aria-label attribute.
+
+**Example: A decorative icon with an adjacent text**
+
+![](/post-images/accessibility/decorative-icon.png)
+
+**Example: A decorative illustration that makes the page more attractive**
+
+![](/post-images/accessibility/decorative-illustration.png)
```

Comment:
We could get these images to show in Federalist preview by prepending the base URL:

```suggestion
**Example: A decorative icon with an adjacent text**

![]({{ '/post-images/accessibility/decorative-icon.png' | absolute_url }})

**Example: A decorative illustration that makes the page more attractive**

![]({{ '/post-images/accessibility/decorative-illustration.png' | absolute_url }})
```

Reply:
Thank you!

Reply:
The images still aren't showing for me in the Federalist preview, though it's not really clear why, since the URL appears to be the one I would expect it to be 🤔 

https://federalist-proxy.app.cloud.gov/site/18f/identity-style-guide/preview/18f/identity-style-guide/je-lg3156-add-accessibility/post-images/accessibility/decorative-illustration.png

Reply:
I'm used it seeing `{{ 'url' | prepend: site.baseurl }}`?

Reply:
Oh I think it's missing `docs/` in the path

Reply:
> I'm used it seeing `{{ 'url' | prepend: site.baseurl }}`?

Not sure 🤷  My understanding is that older versions of Jekyll relied on the developer to do a lot of the manual labor of prepending `baseurl`, and that newer version (4+) try to handle this more automatically in most cases. As best I can tell, `relative_url` and `absolute_url` always had this behavior though.

Somewhat related: https://jekyllrb.com/news/2019/08/20/jekyll-4-0-0-released/#super-powered-content-transformations-

>The `link` and `post_url` tags no longer need `site.baseurl` prepended every time they’re used. Those tags now use our `relative_url` filter to take care of this for you.

---

> Oh I think it's missing `docs/` in the path

I wondered this, but it's a broken link also:

https://federalist-proxy.app.cloud.gov/site/18f/identity-style-guide/preview/18f/identity-style-guide/je-lg3156-add-accessibility/docs/post-images/accessibility/decorative-illustration.png

At least on my local machine, the `dist` folder structure has the image at `/post-images/accessibility/decorative-illustration.png`.

I wasn't sure if maybe the new files weren't being copied into Federalist.

Reply:
`relative_url` seems to be the one. images are showing up both locally and on the preview